index.js

```diff
@@ -155,7 +155,7 @@ module.exports = class OpenSubtitles {
      * @param {Boolean}         extend - Optionnal, fetches metadata from OpenSubtitles
      */
     identify(query) {
-        if (!query && !query.path) throw Error('Missing path')
+        if (!query) throw Error('Missing query')
```

Comment:
The error message should still be "missing path", because what is expected is a "path", the query doesn't matter